include/ipmitool/ipmi_fru.h

```diff
@@ -503,11 +503,11 @@ struct fru_picmgext_amc_p2p_record {
 	unsigned char resource_id         :4;
 	unsigned char /* reserved */      :3;
 	unsigned char record_type         :1;
-#else	
+#else
 	unsigned char record_type         :1;
 	unsigned char /* reserved */      :3;
 	unsigned char resource_id         :4;
-#endif 
```

Comment:
Thanks for cleaning up these hanging spaces.

Reply:
Clang-format handled that -- clang-format handles most of the cleanup in general, even if the end result needs some manual cleanup it's superior at whitespace adjustments.